packages/test-helpers/ganache-cli.sh

```diff
@@ -29,38 +29,30 @@ setup_testing_variables() {
   ACCOUNTS=${ACCOUNTS-200}
 }
 
+rpc_running() {
+  nc -z localhost "$PORT"
+}
+
 start_ganache() {
   echo "Starting ganache-cli..."
-  npx ganache-cli -i ${NETWORK_ID} -l ${GAS_LIMIT} -a ${ACCOUNTS} -e ${BALANCE} -p ${PORT} > /dev/null &
+  node_modules/.bin/ganache-cli -i ${NETWORK_ID} -l ${GAS_LIMIT} -a ${ACCOUNTS} -e ${BALANCE} -p ${PORT} > /dev/null &
```

Comment:
It worked I thought it might be better to use the binary as I saw it in an example. But giving it a second thought having npx allows us to also not include ganache-cli as `peerDependnecy`. What you think about that?